src/main/kotlin/dev/akkinoc/spring/boot/logback/access/LogbackAccessEvent.kt

```diff
@@ -37,6 +41,10 @@ class LogbackAccessEvent(private var source: LogbackAccessEventSource) : IAccess
         return source.timeStamp
     }
 
+    override fun getSequenceNumber(): Long {
```

Comment:
Not sure if this is necessary

Reply:
It seems like a new feature has been added in Logback v1.3.
https://logback.qos.ch/manual/configuration.html#sequenceNumberGenerator

I have made corrections here:
https://github.com/akkinoc/logback-access-spring-boot-starter/pull/336/commits/7bc484d70f0816324873491e90aa47effb114782